sdk/core/abort-controller/package.json

```diff
@@ -67,9 +67,9 @@
     "format": "dev-tool run vendored prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.{ts,cts,mts}\" \"test/**/*.{ts,cts,mts}\" \"*.{js,mjs,cjs,json}\"",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
     "integration-test:browser": "echo skipped",
-    "integration-test:node": "echo skipped",
-    "lint": "eslint package.json api-extractor.json src test",
-    "lint:fix": "eslint package.json api-extractor.json src test --fix --fix-type [problem,suggestion]",
+    "integration-test:node": "dev-tool run test:vitest --no-test-proxy --esm",
```

Comment:
core packages don't have a live test pipeline, when is "integration-test:node" used?

Reply:
When verifying the esm build. I am fine not having it in a pipeline for now.

Reply:
I skipped them for now in https://github.com/Azure/azure-sdk-for-js/commit/56e514b289ff5b3482ce1ffe4666e32a03d1f203.